src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/TestHelper.java

```diff
@@ -140,24 +146,43 @@ public String getRandomFilename(String suffix) {
     ) throws IOException {
         ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
         try (ZipOutputStream zos = new ZipOutputStream(outputStream)) {
+            Map<String, String> tokens = new ConcurrentHashMap<>();
+
             for (String file : files) {
-                zos.putNextEntry(new ZipEntry(file));
+                String dcn = generateDcnNumber();
+                tokens.put("##" + file.replace(".pdf", "") + "##", dcn);
+                zos.putNextEntry(new ZipEntry(dcn + ".pdf"));
```

Comment:
why did the file name change? `files` is no longer files if we no longer use the elements to name files.

Reply:
I'd separate file names from DCNs. They used to be the same by accident or convention, but there's no need for that.